src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1SpaceBookingManagementDomainEventServiceTest.kt

```diff
@@ -428,24 +490,57 @@ class Cas1SpaceBookingManagementDomainEventServiceTest {
 
       val domainEvent = domainEventArgument.captured
 
-      checkDomainEventBookingProperties(domainEvent, spaceBookingWithKeyWorker)
+      checkDomainEventBookingProperties(domainEvent, spaceBookingWithKeyWorker, application.id)
+      val domainEventEventDetails = domainEvent.data.eventDetails
+      assertThat(domainEventEventDetails.previousKeyWorkerName).isEqualTo(previousKeyWorkerName)
+      assertThat(domainEventEventDetails.assignedKeyWorkerName).isEqualTo(keyWorkerName)
+    }
+
+    @Test
+    fun `record keyWorker assigned and emits domain event with offline application`() {
```

Comment:
Is this testing a historical scenario?  Since we aren't expecting any future offline applications?